File: Modulo2/Python/Semana3/ResolucaoPraticas/PI-003/Parte1/Supermercado/menu.py
import os
from controle import inserir_produto, excluir_produto, listar_produtos, consultar_preco

def Menu(produtos):
    while True:
        os.system("cls" if os.name == "nt" else "clear")
        print("=== Sistema de Supermercado ===")
        print("1. Inserir novo produto")
        print("2. Excluir produto cadastrado")
        print("3. Listar todos os produtos")
        print("4. Consultar preço de um produto")
        print("0. Sair")

        opcao = input("Escolha uma opção: ")

        if opcao == "1":
            inserir_produto(produtos)
        elif opcao == "2":
            excluir_produto(produtos)
        elif opcao == "3":
            listar_produtos(produtos)
        elif opcao == "4":
            consultar_preco(produtos)
        elif opcao == "0":
            print("Saindo do programa. Até mais!")
            break
        else:
            print("Opção inválida. Tente novamente.")

File: ProgramacaoImperativa/Semana3/Resolucao-Praticas/PI-013/exercicio3/menu.cpp
#include "menu.h"


void imprimeOpcoes(){
    cout << "------------MENU------------" << endl;
    cout << "1-    Cadastrar novo produto" << endl;
    cout << "2-           Remover produto" << endl;
    cout << "3-  Listar todos os produtos" << endl;
    cout << "4-         Consultar produto" << endl;
    cout << "5 -                     Sair" << endl;
    cout << endl;
    cout << "Escolha : ";
}

void menu(vector<Produto> &produtos, int &quantidadeDeProdutos){

    int escolha;

    while(1){
        imprimeOpcoes();
        cin >> escolha;
        switch (escolha)
        {
        case 1:
            preparaCadastro(produtos,quantidadeDeProdutos);
            break;
        case 2:
            preparaRemocao(produtos,quantidadeDeProdutos);
            break;
        case 3:
            listarTodosProdutos(produtos);
            break;
        case 4:
            consultarProduto(produtos);
            break;
        case 5:
            exit(0);
        default:
            break;
        }
    }

}